docs/format.md

```diff
@@ -0,0 +1,35 @@
+# Format for Operate First Data Science Meetup
+
+## During the session
+
+### Opening and Welcome Note 
+
+Start the session with a warm welcome to all the attendees. Make sure to hit the record button to have a recording for the session.
+
+### Introduction of the Speaker
+
+Introduce the speaker for the session and talk briefly about what the session will be about.
+
+### Presentation
+
+The speaker would be sharing the presentation during this time. 
+
+### Q&A
+
+After the presentation, take questions from the attendees. Make sure all the questions that were asked during/after the session in chat are also answered.
+
+### Closing Note
+
+Thank the speaker and all the attendees for taking out some time to attend the meetup and talk a little about the upcoming session.
+
+## Before the session 
+
+* Send out an email invite to certain target groups, briefing them about the Operate First Data Science Community Meetup.
+* Send out a tweet from the operate-first Twitter account by adding a PR [here](https://github.com/operate-first/operate-first-twitter).
+
+## After the session
+
+* Edit the video recording and post it to Operate First [YouTube Channel.](https://youtube.com/playlist?list=PL8VBRDTElCWqC5WcZUUNJxGn9WslK6llb)
+* Export the list of attendees to google drive.
```

Comment:
is there a particular drive this is being added to?

Reply:
Currently, it gets to Michael's drive since he is the organizer of this event.